resources/configurationFiles/analysisQC.xml

```diff
@@ -45,12 +45,21 @@
         <cvalue name='refDataBaseDirectory' value="/icgc/ngs_share" type="path"/>
 
         <!-- Configuration options -->
-        <cvalue name="runSlimWorkflow" value="true" type="boolean"/>
+        <cvalue name="runSlimWorkflow" value="true" type="boolean"
+                description="Keep this to 'true'. Non-slim scripts are unmaintained."/>
+        <cvalue name="useCombinedAlignAndSampe" value="false" type="boolean"
+                description="Keep this to 'false'. Non-combined BWA scripts are unmaintained."/>
         <cvalue name='useRoddyScratchAsBigFileScratch' value='false' type='boolean'
                 description="If false, all temp-files are written to the place implemented in the scripts (usually close to the result file). Otherwise they go to the node-local temporary directory (e.g. $PBS_SCRATCH_DIR/$PBS_JOBID)."/>
 
         <cvalue name='runFastQC' value='true' type='boolean'/>
         <cvalue name='runCoveragePlots' value='true' type='boolean'/>
+        <cvalue name="runBwaPostAltJs" value="false" type="boolean"
+                description="Whether to run bwa-postalt.js script from bwa.kit. Required for ALT-contig handling. See also K8_VERSION and ALT_FILE."/>
+        <cvalue name="bwaPostAltJsParameters" value="" type="string"
```

Comment:
Have you checked out the "-p HLA_file_prefix" option? The HLA fq files are created in the current working directory, with the workflow they are not copied or created in the destination folder.
For `-p hla_file`, the following files are created in the working directory,
```
hla_file.HLA-A.fq
hla_file.HLA-B.fq
hla_file.HLA-C.fq
hla_file.HLA-DQA1.fq
hla_file.HLA-DQB1.fq
hla_file.HLA-DRB1.fq
```

Reply:
No, I didn't see this. No such files are created in the directory in which I executed Roddy and also not in the RoddyCore directory (the Roddy scripts do a `cd` at some point). 

So, the prefix should be set to the correct path, then. Something like `$outputDir/hla_file`? 

I guess these files are rather small, aren't they? I will add the option to the command.